src/main/java/ch/njol/skript/Skript.java

```diff
@@ -489,6 +489,16 @@ public void onEnable() {
 			classLoadError = e;
 		}
 
+		// Warn about pausing
+		if (Skript.methodExists(Server.class, "getPauseWhenEmptyTime")) {
+			int pauseThreshold = getServer().getPauseWhenEmptyTime();
+			if (pauseThreshold > -1) {
+				Skript.warning("Server pausing is enabled!");
+				Skript.warning("Scripts that interact with the world or entities will not work when the server is paused and may crash your server.");
+				Skript.warning("Set 'pause-when-empty-seconds' to -1 in server.properties to make sure you don't encounter any issues.");
```

Comment:
This line as well implies that if you dont set it to -1 that you WILL encounter issues.

Maybe consider a recommendation:
```suggestion
				Skript.warning("Consider setting 'pause-when-empty-seconds' to -1 in server.properties to make sure you don't encounter any issues.");
```